scripts/box/connect_external_accounts.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+First run 
+    rm -rf website/addons/box/views/
```

Comment:
`box/views` changed from being a folder with several files to a single `views.py`, but the folder and associated `.pyc` files will remain, causing the server to expect `BoxOAuthSettings` even though it's been removed.